### DISCUSSION TITLE
Mechanism for archiving or hiding folders/tags

### DISCUSSION THREAD

**Discussion by oswald3141** [standardnotes/forum#3234] [Suggestions]
I have a bunch of folders created specifically for certain projects. These projects had been completed, and I archived all the notes related to them. I am sure I will not create a note in such a folder anymore. However, the folders are still present in the list on the left side of the main window. I can delete them, but in that case the corresponding tags will be also deleted from the archived notes, and I will not be able to see in the archive to what project a note used to belong.

So, I would like to have some mechanism allowing me to archive a folder, or at least to hide it in the main folders list.

**Comment by Rudtrack** [comment]
Yes, I would love this feature aswell! It's the one thing holding me back from completely switching to Standard Notes as I need a way to archive folders/tags so the notes stay grouped together, but not clutter up either the notes list or the folder list

**Reply by DangerZoneDuchess** [reply]
Wouldn't a simple workaround be to have a tag/folder called "archive" and just move the project folder under it - and keeping the tag collapsed all the time?

**Reply by Rudtrack** [reply]
I tried that for a bit, but after importing all my notes and sorting them correctly, the amount of tags quickly got out of hand. It started being difficult to find the correct one if I wanted to move a note to a new tag for example. So a way to basically keep the tag and its structure, while hiding it and its notes from regular view would be ideal for me!
